tests/metrics/functional/test_object_detection.py

```diff
@@ -0,0 +1,105 @@
+import pytest
+import torch
+
+from pytorch_lightning.metrics.functional.object_detection import mean_average_precision
+
+
+@pytest.mark.parametrize(
```

Comment:
could you make a comment how the different expected values was obtained, as it is not a trivial value to compute

Reply:
Sure! I literally just sat down and calculated the expected values prior to implementing the method and confirmed that the metric returns the same value.